tests/dns/dns-rcode/README.md

```diff
@@ -0,0 +1,5 @@
+Test the `dns.rcode` sticky buffer.
+
+The PCAP here was a request created with Scapy to include answers in
+the request. However the response is from a real DNS server with the
+provided request.
```

Comment:
Please add a reference to the redmine ticket :)

If you do re-use a pcap from a different test, you can indicate that here, too.